components/home/exploreTheWorld/cardDetail.tsx

```diff
@@ -0,0 +1,57 @@
+import Button from '../../button.components'
+
+type AppProps = {
+  price: number;
+  location: string;
+  rooms: number;
+  reviews: number;
+  reviewAmount: number;
+  imgPath: string;
+  hotelName: string;
+  distanceToTownCenter: number;
+}
+
+const DetailedCard = ({price, location, rooms, reviews, reviewAmount, imgPath, hotelName, distanceToTownCenter}: AppProps) => {
+  return (
+    <>
+      <div className="card-wrapper px-4 pt-4 pb-7 bg-white rounded-2xl">
+        <div className="content-wrapper flex flex-col p-0">
+          <div className="top-half flex flex-col justify-center items-start gap-5">
+            <div className="image w-60 h-40 rounded-2xl bg-cover" style={{backgroundImage: `url(${imgPath})`}}></div>
+            <div className="flex flex-row items-center gap-2">
+              <div className="star-icon">
+                <img src="/star.png" alt="star for rating" />
```

Comment:
It seems the problem stems from the img tag, I believe Next requires us to use its own built in Image api.